web/src/app/details/DetailsPage.js

```diff
@@ -30,6 +30,13 @@ const styles = theme => ({
     [theme.breakpoints.up('md')]: { float: 'left' },
     margin: 20,
   },
+  mainHeading: {
+    fontSize: '1.5rem',
+    fontFamily: '"Roboto", "Helvetica", "Arial", sans-serif',
+    fontWeight: 400,
+    lineHeight: 1.33,
+    letterSpacing: '0em',
```

Comment:
same with line weight, height and spacing, unless the defaults are way off when sizing the font down